rpc-lib4j/src/main/java/org/aion/rpc/types/RPCTypes.java

```diff
@@ -981,6 +981,10 @@ public static BlockEnum fromString(String x){
             if(x==null) throw ParseErrorRPCException.INSTANCE;
             if(x.equals("latest")){
                 return LATEST;
+            }else            if(x.equals("pending")){
```

Comment:
Is the no-capital strings as the input STANDARD or should we ignore the case issue?

Reply:
Good catch.